main.py

```diff
@@ -115,6 +115,14 @@ def get_redirect_route(regex_route, handler, defaults=None):
 
     Warning: this method strips off parameters after the trailing slash. URLs
     with parameters should be formulated without the trailing slash.
+
+    Args:
+        regex_route: str. The route to be redirected.
+        handler: function | str. Custom handler method to be called.
```

Comment:
Why function | str? I would expect this to be a function right?

Reply:
Hey @nithusha21 , I added string because, in the documentation, it says the handler can be a callable or a string. see the image below
<img width="1049" alt="Screen Shot 2019-09-22 at 12 55 46" src="https://user-images.githubusercontent.com/26630001/65491730-4dac5880-deb8-11e9-8ba7-2cd5cf7a6d66.png">